pixi.toml

```
@@ -121,6 +125,7 @@ twine = "*"
 typing-extensions = ">=4.6"
 xarray = "*"
 xmipy = "*"
+qgis = "3.34.0.*"
```

```suggestion
qgis = ">=3.28"
```

We generally use * here unless we know some other bound, like we do for QGIS.

---

I changed it to 3.28.*, since that is the LTS release